unit-10/main.js

```diff
@@ -0,0 +1,202 @@
+// Exercise 1: Print out the multiplication table of a number
+function getMultiplyTable(number) {
+  try {
+    if (number == "") {
+      alert("Bạn chưa nhập giá trị");
+    } else if (isNaN(number)) {
+      alert("Giá trị bạn nhập không phải là một số");
+    } else if (number < 1 || number > 9) {
+      alert("Giá trị vừa nhập không thỏa mãn 1 < n <= 9 ");
+    } else {
+      document.write("In bảng nhân: " + number);
+      document.write("<br><br>");
+
+      for (let i = 1; i <= 9; i++) {
+        document.write(number + " x " + i + " = " + number * i);
+        document.write("<br>");
+      }
+    }
+  } catch (ex) {
+    alert(ex);
+  }
+}
+
+// Exercise 2: print out even positive integers
+function getEvenIntegers(number) {
+  try {
+    if (number == "") {
+      alert("Bạn chưa nhập giá trị!");
+    } else if (isNaN(number)) {
+      alert("Giá trị vừa nhập không phải là số");
+    } else if (number < 0 || number > 30) {
+      alert("Giá trị vừa nhập không thỏa mãn 1 <= n <= 30");
+    } else if (number === 1) {
+      document.write("Các số chẵn từ 1 --> " + number + " là ");
+      document.write("Không có số chẵn");
+      document.write("<br>");
+    } else {
+      document.write("Các số chẵn từ 1 --> " + number + " là ");
+      for (let i = 1; i <= number; i++) {
+        if (i % 2 == 0) {
+          document.write(i);
+          document.write("\t");
+        }
+      }
+      document.write("<br><br>");
+    }
+  } catch (error) {
+    alert(error);
+  }
+}
+
+// Exercise 3: Calculate the sum of a positive integer
+function getSumIntegers(number) {
+  try {
+    if (number == "") {
+      alert("Bạn chưa nhập giá trị!");
+    } else if (isNaN(number)) {
+      alert("Giá trị vừa nhập không phải là số");
+    } else if (number < 0 || number > 30) {
+      alert("Giá trị vừa nhập không thỏa mãn 1 <= n <= 30");
+    } else {
+      document.write("Tổng các số từ 1 --> " + number + " là ");
+      let sum = 0;
+      for (let i = 1; i <= number; i++) {
+        sum += i;
+      }
+      document.write(sum);
+      document.write("<br><br>");
+    }
+  } catch (error) {
+    alert(error);
+  }
+}
```

Comment:
```suggestion
const validate = number. => typeof(number) === 'number' && number > 0 && number < 30

function getSumIntegers (number) {
  const isPassedValidate = validate(number)
  if (!isPassedValidate) {
     return 'Failed'
  }
  
  let sum = 0
  for (let i = 0; i < number; i++) {
    sum += i
  }
  
  return sum
}
```

- Tách hàm `validate` để có thể dùng nhiều chỗ
-